lib/model.js

```diff
@@ -2518,7 +2518,7 @@ class Model {
    * @param  {boolean}      [options.hooks=true]             Run before / after bulk create hooks?
    * @param  {boolean}      [options.individualHooks=false]  Run before / after create hooks for each individual Instance? BulkCreate hooks will still be run if options.hooks is true.
    * @param  {boolean}      [options.ignoreDuplicates=false] Ignore duplicate values for primary keys? (not supported by MSSQL or Postgres < 9.5)
-   * @param  {Array}        [options.updateOnDuplicate]      Fields to update if row key already exists (on duplicate key update)? (only supported by MySQL, MariaDB & Postgres >= 9.5). By default, all fields are updated.
+   * @param  {Array}        [options.updateOnDuplicate]      Fields to update if row key already exists (on update of record with existing primary key). Only supported by MySQL, MariaDB & Postgres >= 9.5). Is undefined by default, which results in a SequelizeUniqueConstraintError. To update all fields, use `updateOnDuplicate=Object.keys(Model.attributes)` or `updateOnDuplicate=Object.keys(Model.rawAttributes).filter(attr => attr !== 'createdAt' && attr !== 'updatedAt' && attr !== 'deletedAt')`
```

Comment:
> Is undefined by default

No need for this

> which results in a SequelizeUniqueConstraintError. To update all fields, use `updateOnDuplicate=Object.keys(Model.attributes)` or `updateOnDuplicate=Object.keys(Model.rawAttributes).filter(attr => attr !== 'createdAt' && attr !== 'updatedAt' && attr !== 'deletedAt')`
 
What if their are virtual attributes?

Please use this instead

```
By default no field is updated, which can result in unique constraint error.
```

Reply:
There is no need, except that this issue has already been raised once before and it has been made pretty clear that there would be no in-built support something like updateOnDuplicate='*'. I agree virtual attributes could be a problem, but it would be better to leave a comment about being careful about virtual attributes rather than leaving people high and dry on what is a pretty fundamental use of DBs (upserting many records).